### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by schwabe**
Adding a few template in the hope of getting better bug reports.

Preview:
https://github.com/schwabe/opentx/issues/new/choose

### CODE REVIEW

**Comment by 3djc** [comment]
Maybe the help one could have two links to openrc and rcgroups forums instead ?

**Comment by schwabe** [comment]
Good idea. Made the forums links

**Comment by 3djc** [comment]
👍

File: .github/ISSUE_TEMPLATE/help.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: Help
+about: Problems or question of using OpenTX
+
+--- 
+
+The issue tracker here is for devlopment of OpenTX. This is not a
```
Comment (elecpower) [inline .github/ISSUE_TEMPLATE/help.md:7]: Typo development

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help the OpenTX team to improve OpenTX
+
+---
+*Before submitting,  search the open issue if there is already a report of the bug*
```
Comment (mpaperno) [inline .github/ISSUE_TEMPLATE/bug.md:6]: And search closed ones too... seems like we get quite a few dupes of issues already fixed but not yet released.  In fact I'd probably emphasize searching closed issues in particular.

**Pull request merged**